Formula/g/git-mediate.rb

```diff
@@ -0,0 +1,70 @@
+class GitMediate < Formula
+  desc "Utility to help resolve merge conflicts"
+  homepage "https://github.com/Peaker/git-mediate"
+  url "https://github.com/Peaker/git-mediate/archive/refs/tags/1.0.9.tar.gz"
+  sha256 "ed9b4f5f1ccc295e69d5a368b1b5436968e2290b8e1792a768c0cfae7f91fefb"
+  license "GPL-2.0-only"
+
+  depends_on "ghc@9.2" => :build
```

Comment:
would be good to have an upstream issue linked to this PR about supporting higher ghc, like ghc 9.6